ctapipe/reco/hillas_reconstructor.py

```diff
@@ -165,16 +114,25 @@ class that reconstructs the direction of an atmospheric shower
 
     def __init__(self, subarray, **kwargs):
         super().__init__(subarray=subarray, **kwargs)
-        self._cam_radius_m = {
-            cam: cam.geometry.guess_radius() for cam in subarray.camera_types
+        _cam_radius_m = {
+            cam: cam.geometry.guess_radius().to_value(u.m)
+            for cam in subarray.camera_types
         }
 
         telframe = TelescopeFrame()
-        self._cam_radius_deg = {}
-        for cam, radius_m in self._cam_radius_m.items():
-            point_cam = SkyCoord(0 * u.m, radius_m, frame=cam.geometry.frame)
+        _cam_radius_deg = {}
+        for cam, radius_m in _cam_radius_m.items():
+            point_cam = SkyCoord(0, radius_m, unit=u.m, frame=cam.geometry.frame)
             point_tel = point_cam.transform_to(telframe)
-            self._cam_radius_deg[cam] = point_tel.fov_lon
+            _cam_radius_deg[cam] = point_tel.fov_lon.to_value(u.deg)
```

Comment:
Not directly related to this PR, but is anyone else really disliking this code block and would rather have the cam radius in meter and degree be a property of the camera geometry?

Reply:
We currently have the geometry strictly in one frame and I quite like this...

Reply:
This should rather be replaced by dropping support for doing high-level things in camera frame